app/columns.tsx

```diff
@@ -6,40 +6,55 @@ import { PreviewLinkButton } from "@/components/ui/previewlinkbutton";
 import { ChevronsUpDown, ChevronDown, ChevronRight } from "lucide-react";
 import { PrettyDateRange } from "./dates";
 import { InfoPopover } from "@/components/ui/infopopover";
-
-function getSurfaceTagColor(template: string): string {
-  const surfaceTagColors: any = {
-    aboutwelcome: "bg-red-400",
-    defaultaboutwelcome: "bg-orange-400",
-    feature_callout: "bg-yellow-300",
-    infobar: "bg-lime-300",
-    milestone_message: "bg-green-400",
-    multi: "bg-teal-300",
-    pb_newtab: "bg-sky-400",
-    protections_panel: "bg-blue-500",
-    toast_notification: "bg-indigo-400",
-    toolbar_badge: "bg-purple-400",
-    spotlight: "bg-pink-400",
-    update_action: "bg-rose-400",
-  };
-
-  if (template in surfaceTagColors) {
-    return surfaceTagColors[template];
-  }
-
-  return "";
-}
+import { getSurfaceDataForTemplate } from "@/lib/messageUtils";
 
 function SurfaceTag(template: string, surface: string) {
+  const tagColors = [
```

Comment:
instead of defining the tagColors here, we can fetch them out of SurfaceData (along with the docs) like this:

`let { tagColor, docs } = getSurfaceDataForTemplate(template) }`